client/app/lib/addToRecentsAtLocalStorage.js

```diff
@@ -0,0 +1,37 @@
+function removeMostOutdatedItemIfTypeLimit(list, objectType) {
```

Comment:
IMHO this file looks rather like a service, so maybe move it to `services` folder (+ maybe wrap all these function into some class?)

Reply:
As i told to @gabrieldutra i was confused about the project folder structure but for sure it can be an service class so we can have the implementation details in a single place like you said in the other comment. I will implement it!

Reply:
It's totally not a problem, sorry if I was impolite. I just wanted to suggest in idea how it could be improved :slightly_smiling_face:

Reply:
Its ok you didn't sound impolite, thank you so much for the guidance about the project folder structure! Sorry if i was impolite hahaha i really liked your idea :))